charts/workflow/values.yaml

```diff
@@ -165,6 +165,15 @@ registry-token-refresher:
 
 router:
   dhparam: ""
+  # Any custom router annotations(https://github.com/deis/router#annotations)
+  # which needs to applied can be specified as key values pair under this deployment_annotations key
+  deployment_annotations:
+    #<example-key>: <example-value>
+
+  # Any custom annotations for k8s services like http://kubernetes.io/docs/user-guide/services/#ssl-support-on-aws
+  # which needs to applied can be specified as key values pair under this service_annotations key
```

Comment:
Same as above:
`which need to be applied can be specified as key-value pairs under "service_annotations"`